### PR TITLE
feat: add kafka consumer docs (e5.0)

### PR DESCRIPTION
**Pull request by thalesmg**
https://emqx.atlassian.net/browse/EMQX-8407

- [ ] TODO: add ZH translation after EN version is approved.

### CODE REVIEW

File: zh_CN/data-integration/data-bridge-kafka-consumer.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+     | --------- | -------------------------------------------------- |
+     | `headers` | 包含字符串键-值对的对象                            |
+     | `key`     | Kafka 消息键（由选择的键编码）                     |
+     | `offset`  | Kafka 主题分区中消息的偏移量                       |
+     | `topic`   | Kafka 源主题                                       |
+     | `ts`      | 消息时间戳                                         |
+     | `ts_type` | 消息时间戳类型,  `create`, `append` or `undefined` |
+     | `value`   | Kafka 消息值 (由选择的值编码)                      |
+
+   - **MQTT 载荷模板**的默认值为`${.}`，其中包括编码为 JSON 对象的所有可用数据。例如，选择`${.}`作为模板将会产生以下 Kafka 消息内容：
```
Comment (wivwiv) [inline zh_CN/data-integration/data-bridge-kafka-consumer.md]:
```suggestion
   - **MQTT Payload Template**的默认值为`${.}`，其中包括编码为 JSON 对象的所有可用数据。例如，选择`${.}`作为模板将会产生以下 Kafka 消息内容：
```

File: zh_CN/data-integration/data-bridge-kafka-consumer.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+      "ts_type": "create",
+      "ts": 1679665968238,
+      "topic": "my-kafka-topic",
+      "offset": 2,
+      "key": "key",
+      "headers": {"header_key": "header_value"}
+     }
+     ```
+
+     可以使用点符号进入 Kafka 消息的子字段。例如：`${.value}` 将解析为 Kafka 消息的值，`${.headers.h1}` 将解析为 Kafka `h1` 标题的值（如果存在）。缺失的值将被替换为空字符串。
```
Comment (wivwiv) [inline zh_CN/data-integration/data-bridge-kafka-consumer.md]:
```suggestion
     可以使用点符号选择 Kafka 消息的子字段。例如：`${.value}` 将解析为 Kafka 消息的值，`${.headers.h1}` 将解析为 Kafka `h1` 标题的值（如果存在）。缺失的值将被替换为空字符串。
```

File: zh_CN/data-integration/data-bridge-kafka-consumer.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+     :::tip
+
+     每一对 Kafka 和 MQTT 主题之间的映射关系只能有一个唯一的 Kafka 主题名称，即 Kafka 主题名称不能出现在一对以上的映射关系中。
```
Comment (wivwiv) [inline zh_CN/data-integration/data-bridge-kafka-consumer.md:73]: See how to tell more clearly @LenaLenaPan

File: zh_CN/data-integration/data-bridge-kafka-consumer.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+     :::
+
+     <img src="./assets/bridge-configure.png" alt="bridge-configure" style="zoom:67%;" />
+
+   5. 点击**创建**，将被提示是否使用该数据桥接创建规则。这将允许进一步转换和过滤需要匹配规则的 Kafka 消息，然后转发到其他规则操作，如不同的桥接。有关创建规则的更多信息，请参阅[规则引擎](./rules.md)。
```
Comment (wivwiv) [inline zh_CN/data-integration/data-bridge-kafka-consumer.md]:
```suggestion
   5. 点击**创建**，将提示是否使用该数据桥接创建规则。这可以通过规则进一步处理 Kafka 消息，然后再发送到 MQTT 客户端。有关创建规则的更多信息，请参阅[规则引擎](./rules.md)。
```

File: zh_CN/data-integration/data-bridge-kafka-consumer.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+      :::tip 提示
+
+      创建关联的规则并不是必需的。在**主题映射**中定义的 MQTT 主题将会在没有进一步配置的情况下发布消息。
```
Comment (wivwiv) [inline zh_CN/data-integration/data-bridge-kafka-consumer.md]:
```suggestion
创建关联的规则不是必需的。没有规则的情况下消息也可以发布到 **主题映射** 中设置的 MQTT 主题。
```

**Review by wivwiv — CHANGES_REQUESTED** [review]

File: en_US/data-integration/data-bridge-kafka-consumer.md [hunk 6]
```diff
@@ -20,7 +20,6 @@ The following steps guide you to create data bridge to Kafka in consumer mode.
 
 - Knowledge about EMQX data integration [rules](./rules.md)
 - Knowledge about [data bridge](./data-bridges.md)
-- Relevant Kafka topics should be created before creating the data bridge.
```
Comment (thalesmg) [inline en_US/data-integration/data-bridge-kafka-consumer.md]: I think this is still relevant for this bridge.

**Comment by thalesmg** [comment]
Should the `data-bridge-kafka-consumer.md` files be deleted?

File: en_US/data-integration/data-bridge-kafka-consumer.md (hunk 6 above)
Comment (Meggielqk) [reply en_US/data-integration/data-bridge-kafka-consumer.md]: It is mentioned in the "Create Kafka Topics" section, before "Create Kafka Data Bridge".

**Pull request closed without merging**